### PR TITLE
Adding examples on using revision tags for environments

### PR DESCRIPTION
**Pull request by IaroslavTitov**
### Summary
- Adding examples on using revision tags for environments

### Testing
- All the examples automatically run as tests

### CODE REVIEW

File: examples/py-environments/__main__.py [hunk 1]
```diff
@@ -15,3 +16,26 @@
...
+	organization=environment.organization,
+	environment=environment.name,
+	tag_name=environment.revision.apply(
+    lambda revision: "v" + str(revision)
+  ),
+	revision=environment.revision,
+  opts=ResourceOptions(
+    retain_on_delete=True
+  )
+)
```
Comment (komalali) [inline examples/py-environments/__main__.py:41]: nit: newline
```suggestion
)

```

File: examples/ts-environments/index.ts [hunk 2]
```diff
@@ -12,3 +12,21 @@ var environment = new service.Environment("testing-environment", {
...
+
+// A tag that will be placed on each new version, and remain on old revisions
+var versionTag = new service.EnvironmentVersionTag("VersionTag", {
+  organization: environment.organization,
+  environment: environment.name,
+  tagName: environment.revision.apply((rev: number) => "v"+rev),
+  revision: environment.revision
+}, {
+  retainOnDelete: true
+})
```
Comment (komalali) [inline examples/ts-environments/index.ts:32]:
```suggestion
})

```

File: examples/yaml-environments/Pulumi.yaml [hunk 3]
```diff
@@ -8,9 +8,25 @@ resources:
...
+      revision: ${testing-environment.revision}
+  versionTag:
+    type: pulumiservice:EnvironmentVersionTag
+    properties:
+      organization: ${testing-environment.organization}
+      environment: ${testing-environment.name}
+      tagName: v${testing-environment.revision}
+      revision: ${testing-environment.revision}
+    options:
+      retainOnDelete: true
```
Comment (komalali) [inline examples/yaml-environments/Pulumi.yaml:32]:
```suggestion
      retainOnDelete: true

```

File: examples/ts-environments/index.ts (hunk 2 above)
Comment (komalali) [inline examples/ts-environments/index.ts:32]: You had mentioned yesterday that this resource worked without the outputs... but we're not really using outputs in any of these examples so they're not really tested. Something like `export const tag = versionTag.tagName;` would test it.

**Review by komalali — COMMENTED** [review]
These haven't been added to the test files... was that intentional?

Ah I just realized these are additions to existing examples

**Comment by IaroslavTitov** [comment]
> These haven't been added to the test files... was that intentional?

I didn't really add new tests, just extended existing ones, so there's nothing to add really

**Review by komalali — APPROVED** [review]

File: examples/ts-environments/index.ts (hunk 2 above)
Comment (IaroslavTitov) [reply examples/ts-environments/index.ts]: Yeah, I realized that I never actually used any properties. I tested and checked that after adding to the schema the properties have appeared. (It's easy to check in Dotnet)

**Pull request merged**